core/server/master/src/main/java/alluxio/master/file/FileSystemMasterAuditContext.java

```diff
@@ -140,6 +153,7 @@ public void close() {
     if (mAsyncAuditLogWriter == null) {
       return;
     }
+    mCreationTime = System.nanoTime() - mCreationTime;
```

Comment:
ok i see the confusion. we shall have two properties here
one is `startTime` (or `creationTime`, up to you) and one is `executionTime` like what you had before.
`mCreationTime = System.nanoTime() - mCreationTime;` is simply confusing. 

```suggestion
    mExecutionTime = System.nanoTime() - mCreationTime;
```